tests/wolfram_alpha_test.py

```diff
@@ -0,0 +1,12 @@
+from plugins.wolfram_alpha import WolframAlpha
+
+import vcr
+
+pytest_plugins = ['errbot.backends.test']
+extra_plugin_dir = 'plugins'
+
+my_vcr = vcr.VCR(match_on=['method', 'scheme', 'host', 'port', 'path'])
```

Comment:
To not make the http requests. It is not a good idea to make real requests in a test, and also, this needs an API key and encrypting and decrypting might be an Overkill